test/base/SpecBase.scala

```diff
@@ -117,6 +119,15 @@ trait SpecBase
   val traderDetailsWithConfirmation: TraderDetailsWithConfirmation =
     TraderDetailsWithConfirmation(traderDetailsWithCountryCode)
 
+  val testEoriDetailsSummary: IndividualEoriDetailsSummary = IndividualEoriDetailsSummary(SummaryList())
+  val testApplicantSummary: IndividualApplicantSummary     = IndividualApplicantSummary(SummaryList())
+  val testDetailsSummary: DetailsSummary                   = DetailsSummary(SummaryList())
+  val testMethodSummary: MethodSummary                     = MethodSummary(SummaryList())
+
+  // Use the instances to create an instance of ApplicationSummary
```

Comment:
Please remove this comment. It is not needed as it is obvious like the others in this file, thanks.